config/fi_provider.m4

```diff
@@ -53,7 +53,7 @@ dnl
 AC_DEFUN([FI_PROVIDER_SETUP],[
 	AC_MSG_NOTICE([*** Configuring $1 provider])
 	AC_ARG_ENABLE([$1],
-	      [AS_HELP_STRING([--enable-$1],
+	      [AS_HELP_STRING([--enable-$1@<:@=yes|no|auto|dl|PATH@:>@],
 			      [Enable $1 provider @<:@default=auto@:>@])
```

Comment:
Should we also update this help message to explain what the different options mean?